dotCMS/src/main/java/com/dotmarketing/portlets/folders/business/FolderFactory.java

```diff
@@ -61,6 +61,15 @@ protected Folder findFolderByPath(String path, Host host) throws DotDataExceptio
 		return null;
 	}
 
+	/**
+	 *
+	 * @param folder
+	 * @return
+	 * @throws DotStateException
+	 * @throws DotDataException
+	 *
+	 * @deprecated use {@link #getSubFoldersTitleSort(Folder)}
+	 */
 	protected List<Folder> getSubFolders(Folder folder) throws DotHibernateException, DotStateException, DotDataException {

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @Deprecated annotation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)